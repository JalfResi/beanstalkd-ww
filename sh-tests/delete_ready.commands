put 0 0 0 0

delete 1
quit
