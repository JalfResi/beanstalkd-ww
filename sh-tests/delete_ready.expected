INSERTED 1
DELETED
